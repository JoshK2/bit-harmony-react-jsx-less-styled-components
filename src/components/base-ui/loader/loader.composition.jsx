import React from 'react';
import { Loader } from './loader';

export const BasicLoader = () => {
  return <Loader />;
};
