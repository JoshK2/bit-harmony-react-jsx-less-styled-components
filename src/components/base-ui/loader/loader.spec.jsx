import React from 'react';
import testRenderer from 'react-test-renderer';
import { BasicLoader } from './loader.composition';

describe('Loader component', () => {
  it('renders correctly', () => {
    testRenderer.create(<BasicLoader />);
  });
  it('snapshot test', () => {
    const component = testRenderer.create(<BasicLoader />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
