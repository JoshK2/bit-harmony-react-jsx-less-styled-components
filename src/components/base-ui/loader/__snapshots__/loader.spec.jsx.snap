// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component snapshot test 1`] = `
<div
  style={Object {}}
>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
  <div>
    <div
      style={
        Object {
          "background": "#010101",
        }
      }
    />
  </div>
</div>
`;
